src/screens/OtherProfile/index.js

```diff
@@ -652,14 +652,14 @@ const styles = StyleSheet.create({
   wrapImageProfile: {
     marginTop: 24,
     flexDirection: 'column',
-    backgroundColor: 'red'
+    backgroundColor: COLORS.redalert
   },
   nameProfile: {
     fontFamily: fonts.inter[800],
     fontWeight: 'bold',
     fontSize: 14,
     lineHeight: 17,
-    color: colors.black
+    color: COLORS.black
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color `COLORS.redalert` used in line 655 seems to be a significant change from the previous color 'red'. Ensure that this color change is intentional and aligns with the overall design theme.



<!-- This is an auto-generated comment by OSS CodeRabbit -->